.changeset/olive-dots-whisper.md

```diff
@@ -0,0 +1,5 @@
+---
+"@medusajs/medusa": patch
+---
+
+feat(nested-categories): added admin delete category endpoint
```

Comment:
Not important now, but for future reference, the scope in changesets should be coupled to the version it bumps. So here: `feat(medusa)` 

Another example:
```
// changeset
---
"@medusajs/medusa": patch
"medusa-payment-stripe": patch
---

feat(medusa, medusa-payment-stripe): Something
```

Hope it makes sense :)